package/kedro_viz/launchers/cli.py

```diff
@@ -158,7 +164,6 @@ def run(
             "extra_params": params,
         }
         if autoreload:
-            project_path = Path.cwd()
             run_server_kwargs["project_path"] = project_path
             run_process_kwargs = {
                 "path": project_path,
```

Comment:
The above process calls `run_server -> load_and_populate_data -> load_data` . In `load_data`  we again call bootstrap_project (line 91) which is a duplicate call. Though I like the idea of having bootstrap_project inside data_loader which makes sense considering it loads metadata, we need to remove this if we go by the approach in this PR. For the dev mode we already have bootstrap_project(project_path) in `server.py -> main` at line 143